app/components/primer/spinner_component.html.erb

```diff
@@ -1,7 +1,15 @@
-<span role="status">
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
   <span class="sr-only">Loading</span>
-  <%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <%= render Primer::BaseComponent.new(
+    tag: :svg,
+    viewBox: '0 0 16 16',
+    fill: 'none',
+    animation: :rotate,
+    width: @size,
+    height: @size,
+    vertical_align: :bottom
```

Comment:
Added this because imgs and svgs act weirdly inline without it, and there's no case where we wouldn't want a spinner to only take up its box. Could be too big of a change for this PR?

<img width="924" alt="Screenshot 2021-11-29 at 17 14 17" src="https://user-images.githubusercontent.com/1901935/143913175-536cbf71-1a74-4453-861e-2ab9c5fdffad.png">
<img width="925" alt="Screenshot 2021-11-29 at 17 13 49" src="https://user-images.githubusercontent.com/1901935/143913271-52186303-44d5-4e48-b2fd-834e690e33a2.png">